<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addDialog</class>
 <widget class="QDialog" name="addDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>331</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTextEdit" name="titleEdit">
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="authorEdit">
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="publisherEdit">
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="createdEdit">
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="storedEdit">
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="rendEdit">
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="insertButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Insert to table</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>101</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>true</underline>
        <strikeout>false</strikeout>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <underline>true</underline>
        <strikeout>false</strikeout>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Author</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <underline>true</underline>
        <strikeout>false</strikeout>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Publisher</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Created</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <underline>true</underline>
        <strikeout>false</strikeout>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Stored</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Rent</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Update to table</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
